browser/ui/views/frame/brave_browser_non_client_frame_view_mac.mm

```diff
@@ -100,3 +100,16 @@
       FROM_HERE, base::BindOnce(&views::Widget::ResetWindowControlsPosition,
                                 frame()->GetWeakPtr()));
 }
+
+gfx::Size BraveBrowserNonClientFrameViewMac::GetMinimumSize() const {
+  if (tabs::features::ShouldShowVerticalTabs(browser_view()->browser())) {
```

Comment:
Q) Why only do this when vertical tab is shown?

Reply:
I wanted to omit only tab strip region view part from upstream implementation.

```
gfx::Size BrowserNonClientFrameViewMac::GetMinimumSize() const {
  gfx::Size client_size = frame()->client_view()->GetMinimumSize();
  if (browser_view()->browser()->is_type_normal())
    client_size.SetToMax(
        browser_view()->tab_strip_region_view()->GetMinimumSize());  << here

  // macOS apps generally don't allow their windows to get shorter than a
  // certain height, which empirically seems to be related to their *minimum*
  // width rather than their current width. This 4:3 ratio was chosen
  // empirically because it looks decent for both tabbed and untabbed browsers.
  client_size.SetToMax(gfx::Size(0, (client_size.width() * 3) / 4));

  return client_size;
}
```

Reply:
Updated PR to set min height which is 3/ 4 of width.